src/main/target/OMNIBUSF4/target.h

```diff
@@ -174,6 +174,10 @@
 
 #define AVOID_UART1_FOR_PWM_PPM
 
+#define SPEKTRUM_BIND
+// USART1 RX
+#define BIND_PIN                PA10
```

Comment:
Why not use the symbolic name `UART1_RX_PIN` for a bit more self documenting code. Making the comment obsolete.
And does this really work, is it tested? Isn't there a Sbus inverter in the way, blocking binding. Binding needs a bi-directional Rx data line. I would use `UART3_RX_PIN` for Spektrum Satellite in that case....